Mathlib/NumberTheory/Bertrand.lean

```diff
@@ -70,25 +70,59 @@ theorem real_main_inequality {x : ℝ} (n_large : (512 : ℝ) ≤ x) :
     mul_div 2 x, mul_div_left_comm, ← mul_one_sub, (by norm_num1 : (1 : ℝ) - 2 / 3 = 1 / 3),
     mul_one_div, ← log_nonpos_iff (hf' x h5), ← hf x h5]
   have h : ConcaveOn ℝ (Set.Ioi 0.5) f := by
-    refine'
```

Comment:
I think we need a porting note here noting that this proof was essentially rewritten (because it was too slow?)